tests/test_cfg_end_to_end_tests.py

```diff
@@ -175,99 +84,72 @@ def get_test_cases():
         (f"{DATASET_CONFIGS_ROOT}/kazakh/ksc2/config.yaml", data_check_fn_ksc2),
     ]
 
-
 def check_e2e_test_data() -> bool:
-    """Checks if required environment variables are defined for e2e data.
-
-    Either TEST_DATA_ROOT needs to be defined or both AWS_SECRET_KEY
-    and AWS_ACCESS_KEY.
-    """
-    if os.getenv("TEST_DATA_ROOT"):
-        return True
-    if os.getenv("AWS_SECRET_KEY") and os.getenv("AWS_ACCESS_KEY"):
-        return True
-    return False
+    return bool(os.getenv("TEST_DATA_ROOT") or (os.getenv("AWS_SECRET_KEY") and os.getenv("AWS_ACCESS_KEY")))
 
-
-def get_e2e_test_data_path() -> str:
-    """Returns path to e2e test data (downloading from AWS if necessary).
-
-    In case of downloading from AWS, will create "test_data" folder in the
-    current folder and set TEST_DATA_ROOT automatically (used by the sdp code
-    to locate test data).
-    """
+def get_e2e_test_data_path(rel_path_from_root: str) -> str:
     test_data_root = os.getenv("TEST_DATA_ROOT")
-    if test_data_root:  # assume it's present locally
```

Comment:
add back the comment, it's somewhat useful?

Reply:
Just trying to fix ci/cd, this will be reverted, no worries

Reply:
Done